src/ToolboxBundle/Twig/Extension/GoogleAPIKeysExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ToolboxBundle\Twig\Extension;
+
+use Symfony\Component\HttpFoundation\RequestStack;
+use ToolboxBundle\Manager\ConfigManager;
+use ToolboxBundle\Manager\ConfigManagerInterface;
+
+class GoogleAPIKeysExtension extends \Twig_Extension
+{
+    protected $requestStack;
+
+    /**
+     * JavascriptExtension constructor.
+     *
+     * @param RequestStack $requestStack
+     */
+    public function __construct(RequestStack $requestStack)
+    {
+        $this->requestStack = $requestStack;
+    }
+
+    /**
+     * @return array|\Twig_Function[]
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_Function('toolbox_get_google_map_api_key', [$this, 'getGoogleMapAPIKey'])
+        ];
+    }
+
+    /**
+     * @return mixed
+     *
+     * @throws \Exception
+     */
+    public function getGoogleMapAPIKey()
+    {
+        /** @var ConfigManagerInterface $configManager */
+        $configManager = \Pimcore::getContainer()->get(ConfigManager::class);
```

Comment:
Why not using dependency-injection instead of fetching the service via Container? (afaik, it's more common to get a service via DI and resolving directly from the container is/will be deprecated)